libnmstate/schema.py

```diff
@@ -67,3 +67,20 @@ class LinuxBridge(object):
     PORT_STP_PRIORITY = 'stp-priority'
     PORT_STP_HAIRPIN_MODE = 'stp-hairpin-mode'
     PORT_STP_PATH_COST = 'stp-path-cost'
+
+
+class OvsBridge(object):
```

Comment:
Seems it is either OVS or OvS but not Ovs.

Reply:
It can be any of the 3, I see no problem here. Even libnm uses the same.
But if you commented on this, I guess it is important to you... changing.